README.md

```diff
@@ -435,7 +435,11 @@ To learn more about making a contribution to Pytorch, please see our [Contributi
 
 PyTorch is a community-driven project with several skillful engineers and researchers contributing to it.
 
-PyTorch is currently maintained by [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) with major contributions coming from hundreds of talented individuals in various forms and means.
+PyTorch has adopted a [hierarchical technical governance structure.](https://github.com/pytorch/pytorch/blob/master/docs/source/community/governance.rst)
+
+### Thank You
+
+PyTorch would like to recognize Emeritous maintainer efforts comming from [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) as well as major contributions coming from hundreds of talented individuals in various forms and means.
```

Comment:
nit: just passing by to mention some typos
```suggestion
PyTorch would like to recognize Emeritus maintainer efforts coming from [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) as well as major contributions coming from hundreds of talented individuals in various forms and means.
```

Reply:
thx addressed..